assets/js/vue/components/research-group/OverviewTab.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+    <div>
+        <h4>{{ overview.name }}</h4>
+        <hr>
+        <h6 class="overview-label">Project Director: </h6>
```

Comment:
Use class (add it) `font-weight-bold` (if you want Bootstrap to control bold weight) or just add <b> tags.
Instead of styling later in the file, I fill that way you can see right away this needs to be bold.

Reply:
Forgot about the bootstrap class, good catch!